bin/setup

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
-require 'fileutils'
```

Comment:
why was the alpha order changed?

Reply:
In this PR `bin/bundler` is generated by bundler
But also the rest of binstubs that rails generates have been updated: `bin/rails app:update:bin`
Looks like `bin/setup` has changed a bit.
I's an autogenerated file

Reply:
maybe add it back? don't we need those for setting up the development environment?

Reply:
@aapomm you are right!
Restored copying  `Gemfile.plugins`, `secrets.yml`, `database.yml`